.github/workflows/build_extension_deb.yml

```diff
@@ -0,0 +1,108 @@
+name: Build Extension .deb
+
+defaults:
+  run:
+    shell: bash
+    working-directory: ./
+
+on:
+  release:
+    types:
+      - created
+
+jobs:
+  build-deb:
+    # source: https://github.com/supabase/pg_jsonschema/blob/29fcd5b23abe3dd6a78a490082f34113de7813a1/.github/workflows/release.yml
+    name: release .deb
+    if: github.event_name == 'release'
+    strategy:
+      matrix:
+        extension_name:
+          - pgmq
+        package_name:
+          - pgmq
+        pgrx_version:
+          - 0.11.3
+        postgres: 
+            - 14
+            - 15
+            - 16
+        box:
+          - { runner: ubuntu-20.04, arch: amd64 }
+          - { runner: arm-runner, arch: arm64 }
+    runs-on: ${{ matrix.box.runner }}
+    timeout-minutes: 90
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: build release artifacts
+        run: |
```

Comment:
I personally strongly prefer to put stuff like this into a Shell script that `run` just calls, so it can be tested and more easily modified without having to wait for a slow-ass workflow to finish.

Reply:
Ya that's a good idea